sample_code/ep116/README.md

```diff
@@ -0,0 +1,37 @@
+# [python: .sort() vs sorted(...) (beginner - intermediate)](https://youtu.be/JZYWtmxaCeM)
+
+Today I talk about the difference between list.sort() and sorted(iterable) and when you should pick one over the other (I also show how sorted(...) works under the hood!)
+
+## Interactive examples
+
+### Python
+
+```python
+lst = [1, 5, 34, 2]
```

Comment:
Was that a vague reference to some sort of famous sequence like the fibonacci sequence?